style.css

```diff
@@ -26,6 +27,12 @@ h2 {
   line-height: 22px;
 }
 
+.job-title {
+  align-items: center; 
+  display: flex;
+  margin-top: 4px;
```

Comment:
div(MR_GRID_UNIT, 4) :D

Reply:
Error: Too much framework. Does not compute.